common/Environments/Models/FailedCreateComputeSystemOperation.cs

```diff
@@ -0,0 +1,35 @@
+﻿// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT License.
+
+using System;
+using System.Threading.Tasks;
+using Microsoft.Windows.DevHome.SDK;
+using Windows.Foundation;
+
+namespace DevHome.Common.Environments.Models;
+
+/// <summary>
+/// Class that represents a failed create compute system operation. We use this when we were unable to get the original
+/// ICreateComputeSystemOperation object from the extension. E.g a COM exception was thrown.
+/// </summary>
+public class FailedCreateComputeSystemOperation : ICreateComputeSystemOperation
+{
+    public FailedCreateComputeSystemOperation(Exception exception, string localizedErrorMessage)
+    {
+        LocalizedErrorMessage = localizedErrorMessage;
+        Exception = exception;
+    }
+
+    public string LocalizedErrorMessage { get; }
+
+    public Exception Exception { get; }
+
+    public event TypedEventHandler<ICreateComputeSystemOperation, CreateComputeSystemProgressEventArgs> Progress = (s, e) => { };
```

Comment:
This hurts my eyes as it's completely unnecessary.

Reply:
Interestingly stylecop is the reason for this one. I think it might be time to standardize which stylecop errors we care about or not in Dev Home.